pyproject.toml

```diff
@@ -96,3 +96,33 @@ ignore = [
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tool.pixi.dependencies]
+gymnasium = ">=0.29.1"
+loop-rate-limiters = ">=1.0.0"
+msgpack-python = ">=1.0.2"
+numpy = ">=1.24.3"
+scipy = ">=1.10.0"
+upkie_description = ">=2.1.0"
+
+[tool.pixi.pypi-dependencies]
+upkie = { path = ".", editable = true }
+
+[tool.pixi.feature.docs.dependencies]
+python = "=3.12"
```

Comment:
Did you really need python 3.12 ?
With pyproject.toml, pixi will install python based on `requires-python = ">=3.8"`

Reply:
Oh no that's not needed any more indeed. Good catch! :fishing_pole_and_fish: